runtime-modules/content/src/lib.rs

```diff
@@ -1517,7 +1546,17 @@ decl_module! {
                     new_video_state_bloat_bond));
         }
 
-        #[weight = 10_000_000] // TODO: adjust Weight
+        /// Claim and withdraw reward in JOY from channel account
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (H)` where:
+        /// - `H` is the lenght of the provided merkle `proof`
+        /// - DB:
+        ///    - O(1)
+        /// # </weight>
+        #[weight = WeightInfoContent::<T>::claim_and_withdraw_member_channel_reward(proof.len() as u32).max(WeightInfoContent::<T>::claim_and_withdraw_curator_channel_reward(proof.len() as u32))]
```

Comment:
Respecting max. line length:
```suggestion
        #[weight =
            WeightInfoContent::<T>::claim_and_withdraw_member_channel_reward(proof.len() as u32)
                .max(WeightInfoContent::<T>::claim_and_withdraw_curator_channel_reward(
                    proof.len() as u32
                ))]
```